tests/phpunit/tests/theme/wpThemeJson.php

```diff
@@ -581,7 +581,7 @@ public function test_get_stylesheet() {
 	 * @ticket 60936
 	 * @ticket 61165
 	 */
-	public function test_get_stylesheet_support_for_shorthand_and_longhand_values() {
+	public function test_get_styles_for_block_support_for_shorthand_and_longhand_values() {
```

Comment:
The logic for outputting styles for longhand and shorthand values is in `get_styles_for_block`, so better to test that function directly.